pyproject.toml

```diff
@@ -7,13 +7,12 @@ build-backend = "setuptools.build_meta"
 
 [tool.isort]
 profile = "black"
-extend_skip = [".github", ".idea", "__pycache__", "gui_manager.py", "test_core.py", "test_legacy_core.py", "test_plugin_script.py"]
+extend_skip = [".github", ".idea", "__pycache__"]
```

Comment:
These exclusions are generating the Actions errors on workflow. They're needed as exceptions to isort linting because these files were not updated with these rules.

Reply:
I have updated the files and there are no more lints associated with these